### ISSUE TITLE
iwildcard isn't being case insensitive.

### ISSUE THREAD

**Issue by kylebrandt** [OpenTSDB/opentsdb#658]
The following returns results:

```
{
  "start": "1h-ago",
  "queries": [
    {
      "aggregator": "sum",
      "metric": "nexpose.asset_group.risk_score",
      "rateOptions": {},
      "filters": [
        {
          "type": "wildcard",
          "tagk": "asset_group",
          "filter": "SRE-*",
          "groupBy": true
        }
      ]
    }
  ]
}
```

All results that come back start with "SRE-", where SRE is all caps. When I changed the same query to iwildcard, I get zero results:

```
{
  "start": "1h-ago",
  "queries": [
    {
      "aggregator": "sum",
      "metric": "nexpose.asset_group.risk_score",
      "rateOptions": {},
      "filters": [
        {
          "type": "iwildcard",
          "tagk": "asset_group",
          "filter": "SRE-*",
          "groupBy": true
        }
      ]
    }
  ]
} 
```

**Comment by manolama** [comment]
Well that's a bummer :( Will take a look shortly.

**Comment by manolama** [comment]
Fixed in 53b1959773359a19683e8d4de94ec40915a88b4f. Thanks.
